app/controllers/plan_exports_controller.rb

```diff
@@ -46,6 +46,13 @@ def show
       @selected_phase = @plan.phases.order("phases.updated_at DESC")
                              .detect { |p| p.visibility_allowed?(@plan) }
     end
+
+    #ISSUE205: add contributor to cover page
+    @hash[:data_curation] = Contributor.where(:plan_id => @plan.id).data_curation
+    @hash[:investigation] = Contributor.where(:plan_id => @plan.id).investigation
+    @hash[:pa] = Contributor.where(:plan_id => @plan.id).project_administration
+    @hash[:other] = Contributor.where(:plan_id => @plan.id).other
```

Comment:
We should keep the hash syntax annotation for parameter specificatoin instead of the fat arrow; the rest of the code uses the hash syntax and in general that is what is currently used in ruby.

Reply:
sorry can you give me an example or referred link? I wanted to update the existing @hash, so I was trying a syntax like:

```
h = {foo: 0, bar: 1, baz: 2}
h[:bat] = 3 # => 3
```

Am I on the wrong direction?

Reply:
Certainly, I mean this piece of code:

```Ruby
Contributor.where(:plan_id => @plan.id)
```

Changed to

```Ruby
Contributor.where(plan_id: @plan.id)
```

Reply:
Ohhh I see what you meant. thanks!